templates/_helpers.yaml

```diff
@@ -53,6 +53,32 @@
 
 {{/* User defined Airflow environment variables */}}
 {{- define "custom_airflow_environment" }}
+{{- if .Values.mountAllFromSecretName }}
+envFrom:
+{{- include "custom_airflow_environment_mount_all" . }}
+env:
+{{- include "custom_airflow_environment_config" . }}
+{{- include "standard_airflow_environment" . }}
+{{- else }}
```

Comment:
@kushalmalani does this work right? I thought the last time we talked we determined that one of these - `env` or `envFrom` will clobber the other.

Also, if we are using `mountAllFromSecretName`, we don't want both `custom_airflow_environment_mount_all` and `custom_airflow_environment_config` right? It's the new vs old way of templating the user-supplied env vars in right?

Reply:
@schnie It does work right. The reason for separating them was to make sure it works fine and it only clobbers if we are passing the same env vars via envFrom and env, in which case env takes precedence.

The old way of adding custom env vars is defined in "custom_airflow_environment_mount_single". The config part is common, which adds KubernetesExecutor related environment variables.

Reply:
Oh I see. Should we rename that helper to be kube executor specific to make that clear?

Also, instead of iterating over all the env vars for the kube executor section, I wonder if there is a way to use the `envFrom` syntax. Otherwise we run into similar issues... :thinking:

Reply:
@dimberman know if thats possible?

Reply:
@schnie That helper also adds env vars from .Values.env for non-kube executor part. I could pull that out into a separate helper, but i felt i was maybe over engineering it.

I wondered if i could use envFrom instead but that requires us to give a secret name for it to pull values from and KubernetesExecutor env vars are created with that range. We will need to create a separate secret file for that.